canonical/ubuntu.com
# Enable form validation with Buy button

## Done

- Remove the `disabled` status from the Buy button 
- Display error message for each empty form field after submit

## QA

- Go to /pro/subscribe
- Click buy button 
- Check buy button is always active
- QA with fields and check error messages are displaying properly

## Issue / Card

Fixes #https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/801?modal=detail&selectedIssue=WD-2533
